i18n/tests/test_internationalizable.py

```diff
@@ -10,7 +10,8 @@ class InternationalizableTestCase(TestCase):
     Basic tests for the Internationalizable abstract model
     """
     def setUp(self):
-        TestModel(title="Test Title", description="Test Description").save()
+        TestModel.objects.all().delete()
```

Comment:
Why does this help?

Reply:
Good question! Without this, the second test run with `--keepdb` fails, because the data generated in the first run still exists in the database:
```
./manage.py test i18n.tests --keepdb 
...
======================================================================
FAIL: test_gather_strings (i18n.tests.test_internationalizable.InternationalizableTestCase)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/Users/dsb/src/cb/i18n/tests/test_internationalizable.py", line 24, in test_gather_strings
    "title": u"Test Title"
AssertionError: {'1':[56 chars]tle'}, '3': {'description': u'Test Description[89 chars]le'}} != {'1':[56 chars]tle'}}
- {'1': {'description': u'Test Description', 'title': u'Test Title'},
?                                                                   ^

+ {'1': {'description': u'Test Description', 'title': u'Test Title'}}
?                                                                   ^

-  '2': {'description': u'Test Description', 'title': u'Test Title'},
-  '3': {'description': u'Test Description', 'title': u'Test Title'}}
----------------------------------------------------------------------
```

Reply:
I believe this should instead go [in the `tearDown` method](https://docs.python.org/3/library/unittest.html#unittest.TestCase.tearDown)